src/dftbp/timedep/timeprop.F90

```diff
@@ -3046,7 +3046,16 @@ subroutine writeTDAutotest(this, dipole, energy, deltaQ, coord, totalForce, occ,
     type(TFileDescr) :: fdAutotest
 
     call openFile(fdAutotest, trim(this%autotestTag), mode="a")
-    call taggedWriter%write(fdAutotest%unit, tagLabels%tdenergy, energy%eSCC)
+
+    select case(this%hamiltonianType)
+    case(hamiltonianTypes%dftb)
+      call taggedWriter%write(fdAutotest%unit, tagLabels%tdenergy, energy%eSCC)
+    case(hamiltonianTypes%xtb)
+      call taggedWriter%write(fdAutotest%unit, tagLabels%tdenergy, energy%eSCC + energy%Erep&
+          & + energy%EDisp + energy%EHalogenX)
+    end select
```

Comment:
Is this change really necessary? In the time propagation we would like to look at the SCC energy. For backwards compatibility it might be nice to write out all energies as before, but maybe that is not what we realy want here.